CONTRIBUTING.md

```diff
@@ -113,16 +116,22 @@ correctness.
 doesn't make sense to write templates for. For example, a dataset might just be
 text without any annotations. For other cases, ask on Slack. If skipping a dataset,
 mark it in red on the spreadsheet.
+* **Choosing input and output pairs.** Lots of datasets have multiple columns that can be
+combined to form different (input, output) pairs i.e. different "tasks". Don't hesitate to
+introduce some diversity by prompting a given dataset into multiple tasks and provide some
+description in the "Template Reference" text box. An example is given
+in the already prompted `movie_rationales`.
 
 ## Uploading Templates
 
 Once you save or modify a template, the corresponding file inside the `templates`
 directory in the repo will be modified. To upload it, following these steps:
-1. Run `make style` and `make quality`.
-2. Commit the modified template files (anything under `templates`) to git.
-3. Push to your fork on GitHub.
-4. Open a pull request against `main` on the PromptSource repo.
-5. When the PR is merged into main, mark the dataset in green on the spreadsheet. 
+1.
```

Comment:
@VictorSanh Was there a step you wanted to add here?

Reply:
oh no, i just messed up, corrected it